app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -30,9 +28,6 @@ class MainActivity : AppCompatActivity() {
         lView.adapter = myListAdapter
         if (savedInstanceState == null) {
             setupBottomNavigationBar()
-//            supportFragmentManager.beginTransaction()
-//                .replace(R.id.container, MainFragment.newInstance())
-//                .commitNow()
```

Comment:
Suggestion 1: Removed commented-out code to prevent the code from becoming unnecessarily large.
Source: https://rules.sonarsource.com/kotlin/RSPEC-125 and https://docs.sonarqube.org/display/SONARQUBE45/Not+Enough+or+Too+Many+Comments